src/cycax/cycad/assembly.py

```diff
@@ -134,10 +138,9 @@ def rotateFreezeTop(self, part: CycadPart):
         Args:
             part: This is the part that will be rotated.
         """
-        part.rotate[part.pos["z"]] = (part.rotate[part.pos["z"]] + 90) % 360
+        part.rotate.append(2)
```

Comment:
As mentioned in https://github.com/tsolo-io/cycax/pull/21#discussion_r1251110701

2 here is a magic number. I think it would be a lot better if this was `part.rotate.append({'side': TOP, 'angle': 90})`

Reply:
SS162